imageGenerator/imageGen.py

```diff
@@ -0,0 +1,83 @@
+import json
+import os
+import sys
+
+import requests
+from dotenv import load_dotenv
+from requests.exceptions import RequestException
+
+# stable-diffusion-xl image gen-------------------------------------------------
+
+from diffusers import DiffusionPipeline
+from PIL import Image 
+import torch
+
+pipe = DiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-xl-base-1.0", torch_dtype=torch.float16, use_safetensors=True, variant="fp16")
+pipe.to("cuda")
+
+# --------------------------------------------------------------------
+
+dot = os.path.dirname(os.path.realpath(__file__))
+load_dotenv(f"{dot}/../local.env") # todo: refactor this to check for containerized builds, in which case we should use the `virtual.env` file
+API_PORT = os.getenv("API_PORT")
+API_HOST = os.getenv("API_HOST")
```

Comment:
@gpseal Your image gen module will be using token to access the route. Could you please add the following to your code?
```python
TOKEN = os.getenv("IMAGE_GEN_TOKEN")
```

Reply:
@ArtemKech. since this module is never going to be deployed at this stage, I think security is bottom priority considering the time we have left.